src/picongpu/submit/submitAction.sh

```diff
@@ -18,21 +18,34 @@
 # If not, see <http://www.gnu.org/licenses/>.
 #
 
+## check if input files are newer than the picongpu binary
+numNewerFiles=$(find $TBG_projectPath/include -type f \
```

Comment:
yes should try that to reduce FS load, it's just always a mess to work with multiline bash strings :D